scripts/getsamplesummary.py

```diff
@@ -1,90 +1 @@
-#!/usr/bin/python
-#
-import sys
-import datetime
-import time
-import glob
-import re
-import socket
-import os
-import select
-from access import db, lims
-
-outputdir = '/mnt/hds/proj/bioinfo/tmp/MIPP/exomes/'
-
-fc = "flowcell"
-if len(sys.argv) > 0:
-  try:
-    sys.argv[1]
-  except NameError:
-    sys.exit("Usage: " + sys.argv[0] + " <flowcell name>")
-  else:
-    fc = sys.argv[1]
-else:
-  sys.exit("Usage: " + sys.argv[0] + " <flowcell name>")
-
-params = db.readconfig("non")
-
-fc_samples = {}
-def getsamplesfromflowcell(pars, flwc):
```

Comment:
It might just be me but I find it clearer to read long variable names that use underscores to separate words :smiley: like `get_samples_from_flowcell`.